manifests/plugin/ansible/runner.pp

```diff
@@ -0,0 +1,38 @@
+class foreman_proxy::plugin::ansible::runner {
+  $package = 'ansible-runner'
+
+  case $::osfamily {
+    'Debian': {
+      include ::apt
+      apt::key { 'ansible-release':
+        id      => 'AC48AC71DA695CA15F2D39C4B84E339C442667A9',
+        source  => 'https://releases.ansible.com/keys/RPM-GPG-KEY-ansible-release.pub',
+      }
+      ::apt::source { 'ansible-runner':
+        repos    => 'main',
+        location => 'https://releases.ansible.com/ansible-runner/deb',
+        key      => 'AC48AC71DA695CA15F2D39C4B84E339C442667A9',
+        include  => {
+          src => false,
+        },
+      }
+    }
+    'RedHat': {
+      yumrepo { 'ansible-runner':
+        descr    => "Ansible runner",
+        baseurl  => "baseurl=https://releases.ansible.com/ansible-runner/rpm/epel-7-x86_64/",
+        gpgcheck => true,
+        gpgkey   => "https://releases.ansible.com/keys/RPM-GPG-KEY-ansible-release.pub",
+        enabled  => '1',
+      }
+    }
+    default: {
+      $package = 'python-ansible-runner'
```

Comment:
This will fail because you can't change variables in Puppet. That means you need to define the package name in the various branches